web/styles/input_pill.css

```diff
@@ -45,6 +45,11 @@
             min-width: 0;
             align-items: center;
             margin: 0 5px;
+
+            .zulip-icon-bot {
+                font-size: 12px;
+                margin-left: 3px;
```

Comment:
This `3px` `margin-left` value shows up a lot. Let's put it into a CSS variable, like `--margin-lefthand-bot-icon-offset`, somewhere thoughtful in the `zulip.css` file's stack of variables.